spikeinterface/qualitymetrics/misc_metrics.py

```diff
@@ -488,6 +485,207 @@ def compute_amplitudes_cutoff(waveform_extractor, peak_sign='neg',
 )
 
 
+def compute_amplitudes_median(waveform_extractor, peak_sign='neg'):
```

Comment:
The function name is not intuitive.
I would add the log concept in the title.

Reply:
The return value is not log